controllers/mirroring/mirroring_controller.go

```diff
@@ -286,11 +292,78 @@ func (r *MirroringReconciler) reconcileRbdMirror(clientMappingConfig *corev1.Con
 			r.log.Error(err, "Failed to create/update the CephRBDMirror", "CephRBDMirror", rbdMirror.Name)
 			return true
 		}
+
+		// Enable debug logging for the rbd mirror if it is not already enabled
+		if !rbdMirrorDebugLoggingEnabled {
+			rbdMirrorDebugLoggingJob, err := util.GetRbdMirrorDebugLoggingJob(util.EnableRbdMirrorDebugLoggingJobName, clientMappingConfig.Namespace, util.EnableRbdMirrorDebugLoggingCommands, clientMappingConfig)
+			if err != nil {
+				r.log.Error(err, "Failed to create the enable rbd mirror debug logging job")
+				return true
+			}
+
+			err = r.Client.Create(r.ctx, rbdMirrorDebugLoggingJob)
+			if err != nil && !k8serrors.IsAlreadyExists(err) {
+				return true
+			}
+			// Check if the job has succeeded
+			job := &batchv1.Job{}
+			err = r.Client.Get(r.ctx, types.NamespacedName{Name: util.EnableRbdMirrorDebugLoggingJobName, Namespace: clientMappingConfig.Namespace}, job)
+			if err != nil {
+				return true
+			}
```

Comment:
```suggestion
			if err = r.Client.Get(r.ctx, client.ObjectKeyFromObject(rbdMirrorDebugLoggingJob), job); err != nil {
				return true
			}
```

Reply:
I had raised the same question back in 4.12, At that point it was said to not pollute the ceph config with unrelated settings on non-rdr clusters. I don't know if that argument still holds true.